build.sbt

```diff
@@ -214,6 +200,34 @@ lazy val docs = Project(
     resolvers += Resolver.jcenterRepo,
   )
 
+lazy val pluginTesterScala = Project(
+  id = "akka-grpc-plugin-tester-scala",
+  base = file("plugin-tester-scala")
+)
+  .settings(Dependencies.pluginTester)
+  .settings(commonSettings)
+  .enablePlugins(akka.grpc.NoPublish)
+  .settings(
+    ReflectiveCodeGen.codeGeneratorSettings ++= Seq("flat_package")
+  )
+  .pluginTestingSettings
```

Comment:
I'm not sure we really want to use the 'reflective' codegen for the `plugin-tester` projects: I found it quite nice to be able to test against a 'real' sbt project using the 'real' plugin, both on CI and during development.

Reply:
I came to work on this, because I was reviewing documentation which takes code snippets from `plugin-tester-scala` but I couldn't readily edit those files because IDEA didn't load them correctly because of the project structure (though someone said, that it would work when opening IDEA from the subdirectory).

As you can see in the history, I tried to preserve the old directory structure next to the new one by creating new subdirectories, linking to the same sources but without the standalone sbt build files. 

> found it quite nice to be able to test against a 'real' sbt project using the 'real' plugin, both on CI and during development.

I came to the conclusion that it isn't worth it to keep the extra copy because the plugin-tester projects are no real sbt projects because they still reference the main project for loading the plugin instead of publishing it first.

> On the other hand, during development we can now/soon use the example/quickstart/demo projects for this, so it might not be a big deal anymore.

Exactly.

Reply:
> couldn't readily edit those files because IDEA didn't load them correctly

Yes that is annoying

> still reference the main project for loading the plugin instead of publishing it first

You're right - I thought we had to `publishLocal`/`publishM2` and then use `-Dplugin.version`, but that goes for scripted tests and non-sbt builds, not for these projects.

>> On the other hand, during development we can now/soon use the example/quickstart/demo projects for this, so it might not be a big deal anymore.
> Exactly.

Agreed then!